CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ project(rocm-cmake)
 
 install(DIRECTORY share DESTINATION .)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ROCMConfigVersion.cmake DESTINATION share/rocm/cmake)
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_LIST_DIR}/LICENSE")
+install(FILES ${CPACK_RESOURCE_FILE_LICENSE} DESTINATION share/doc/rocm-cmake)
+set(CPACK_RPM_PACKAGE_LICENSE "MIT")
```

Comment:
Why do we have a default MIT?

Reply:
Not a default, this is the license type for rocm-cmake.

Reply:
Actually, doesnt rom_create_package find the license file automatically?